libtransmission/session.h

```diff
@@ -1100,7 +1114,10 @@ struct tr_session
     // depends-on: open_files_
     tr_torrents torrents_;
 
-    // depends-on: settings_, session_thread_, torrents_
+    // depends-on: settings_, session_thread_, timer_maker_, web_
+    tr_global_ip_cache global_ip_cache_{ *this };
+
+    // depends-on: settings_, session_thread_, torrents_, global_ip_cache_
```

Comment:
Is this comment change correct?  I don't see where WebMediator uses the ip cache..?

Reply:
Oh that change is meant for `tr_web`. It uses `tr_global_ip_cache.global_source_addr()` indirectly by calling `tr_session.publicAddress()`.

What is the correct comment format here?

Reply:
oh I see; yes you're right. Maybe a parenthetical `via tr_session.publicAddress()`?

Reply:
Sure